src/page-modules/departures/details/estimated-call-rows.tsx

```diff
@@ -0,0 +1,199 @@
+import { MonoIcon } from '@atb/components/icon';
+import { Button } from '@atb/components/button';
+import Link from 'next/link';
+import DepartureTime from './departure-time';
+import DecorationLine from './decoration-line';
+import { and } from '@atb/utils/css';
+
+export type EstimatedCallRowsProps = {
+  calls: EstimatedCallWithMetadata[];
+  mode: TransportModeType;
+};
+
+export function EstimatedCallRows({ calls, mode }: EstimatedCallRowsProps) {
+  const { t } = useTranslation();
+  const [collapsed, setCollapsed] = useState(true);
+
+  const passedCalls = calls.filter((c) => c.metadata.group === 'passed');
+  const showCollapsable = passedCalls.length > 1;
+
+  const estimatedCallsToShow = calls.filter(
+    (c) => c.metadata.group !== 'passed',
+  );
+
+  const passedCallsToShow = collapsed ? [passedCalls[0]] : passedCalls;
+
+  const collapseButton = showCollapsable ? (
+    <Button
+      onClick={() => setCollapsed(!collapsed)}
+      title={t(
+        PageText.Departures.details.collapse.label(passedCalls.length - 1),
+      )}
+      mode="transparent"
+      size="compact"
+      icon={{
+        right: collapsed ? (
+          <MonoIcon icon="navigation/ExpandMore" />
+        ) : (
+          <MonoIcon icon="navigation/ExpandLess" />
+        ),
+      }}
+      className={style.collapseButton}
+      buttonProps={{
+        'aria-label': t(PageText.Departures.details.collapse.a11yHint),
+      }}
+    />
+  ) : null;
+
+  return (
+    <div className={style.callRows}>
+      {passedCalls.length > 0 && (
+        <div className={style.callRows__container}>
+          {passedCallsToShow.map((call, i) => (
+            <motion.div
+              key={`${call.quay.id}-${call.aimedDepartureTime}`}
+              initial={{ opacity: 0 }}
+              animate={{ opacity: 1 }}
+              exit={{ opacity: 0 }}
+              transition={{
+                delay: i * 0.05,
+              }}
+            >
+              <EstimatedCallRow
+                call={call}
+                mode={mode}
+                collapseButton={
+                  call.metadata.isStartOfServiceJourney ? collapseButton : null
+                }
+              />
+            </motion.div>
+          ))}
+        </div>
+      )}
+
+      <div className={style.callRows__container}>
+        {estimatedCallsToShow.map((call, i) => (
+          <motion.div
+            key={`${call.quay.id}-${call.aimedDepartureTime}`}
+            initial={{ opacity: 0 }}
+            animate={{ opacity: 1 }}
+            exit={{ opacity: 0 }}
+            transition={{
+              delay: i * 0.015,
+            }}
+          >
+            <EstimatedCallRow call={call} mode={mode} collapseButton={null} />
+          </motion.div>
+        ))}
+      </div>
+    </div>
+  );
+}
+
+type EstimatedCallRowProps = {
+  call: EstimatedCallWithMetadata;
+  mode: TransportModeType;
+  collapseButton: JSX.Element | null;
+};
+function EstimatedCallRow({
```

Comment:
I think an estimated call should have a descriptive a11y label. As it is now it's like "Sanntid 15:01 15:05".

Reply:
It should be "Sanntid 15:01 15:05 Persaunet leir". Are you not getting the quay name? 🤔 This could be improved for sure. It's currently the same as in the app

Reply:
Probably got that. But which one is sanntid for instance.

Reply:
Point is, maybe it should be something more descriptive that describes which is the realtime and which is the expected. "Departure from {stopplace}. Expected {time}, real time {time}" or something.

Reply:
My bad, it should. Fixed now! 
"Sanntid 13:47 Rutetid 13:45"